bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/inverter.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="inverter">
+
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="localConnect"/>
+		</supported-bridge-type-refs>
+
+		<label>Default Inverter Label</label>
+		<description>The inverter representation in OpenHAB</description>
+
+		<channels>
+			<channel id="inverterOutputPower" typeId="system.electric-power"/>
```

Comment:
what do we consider a generic type? I guess the electric current/power is self-explanatory. Can you give me an example which one should be labeled more thoroughly?

Reply:
For example `pv1-voltage` and `pv2-voltage` where you would otherwise not be able to distinguish them by their default labels ("Electric Voltage"):
https://github.com/openhab/openhab-core/blob/f5a0b37b3dfc2bc188c6f80dca96b91a0fd72b6e/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java#L380-L388

Reply:
Oki. Got that... this means probably most of 'em :) I'll do it in the next round of changes.